typescript/sdk/src/deploy/HyperlaneDeployer.ts

```diff
@@ -438,55 +434,40 @@ export abstract class HyperlaneDeployer<
       );
     }
 
-    return new ProxiedContract<C, TransparentProxyAddresses>(
-      implementation.attach(proxy.address) as C,
-      {
-        kind: ProxyKind.Transparent,
-        proxy: proxy.address,
-        implementation: implementation.address,
-      },
-    );
+    return implementation.attach(proxy.address) as C;
```

Comment:
nice

Reply:
we should make sure the verification stuff is output here

Reply:
It looks like we append to verificationInputs in deployContractFromFactory, which is what we use under the hood in this fn, so I think we're good?